js/index.js

```diff
@@ -0,0 +1,30 @@
+// listen to form submit event and make sure page doesn't refresh
+// grab value out of input field
+// make a decision: is that city NYC, if it is, change background, else - if that city is LA, change bg to LA
+$(document).ready(function() {
+
+	$('form').submit(addEntry);
+
+	function addEntry(event) {
+		event.preventDefault();
+		var userInput = $('#city-type').val();
+		console.log(userInput)
+		if (userInput==='austin' || userInput==='Austin') {
+			 $('body').addClass('austin');
+			 }
+	 	if (userInput==='sf' || userInput==='SF') {
```

Comment:
rather than trying to match the case for all possible values, its easier if you just force whatever the user typed in into lowercase (or uppercase) and then just compare against that. 

e.g.

```
var userInput = $('#city-type').val();
userInput = userInput.toLowerCase();  // now "SF", "sf", "Sf", or "sF" would just be "sf"
```
then you could simply write your `if` conditions against the lower case value